InvenTree/common/models.py

```diff
@@ -1371,7 +1371,7 @@ def save(self, *args, **kwargs):
         'PLUGIN_ON_STARTUP': {
             'name': _('Check plugins on startup'),
             'description': _('Check that all plugins are installed on startup - enable in container environments'),
-            'default': settings.DOCKER,
+            'default': False,  # settings.DOCKER,
```

Comment:
@matmair strangely, this change seems to have fixed the issue. Commenting out the line has resulted in 5/5 CI pass results in a row.

Reply:
Intresting! No idea why but let’s keep it that way. We can always enable plugins before tests.